src/main.cpp

```diff
@@ -15,7 +15,9 @@ int main(int argc, char **argv)
 	if (!config.getErrorMessage().empty())
 		return 1;
 	
-	std::cout << config << std::endl; // should be in the DEBUG?
+	//std::cout << config << std::endl; // should be in the DEBUG?
+	Debug::enable(true);
+	Debug::setLevel(Debug::NORMAL);
 	Server webserv(config);
```

Comment:
Since you are adding Debug statements, These two lines should be before the std::cout, and the //std::cout << config << std::endl; // should be in the DEBUG? line should be a debug log with values CONFIG. Probably we need an overload of the function cause we are not passing a string but an object.

Reply:
This is why I could not add it to the Debug class. Lets improve Debug in the next PR